src/include/common/exception.h

```diff
@@ -14,11 +14,12 @@ namespace terrier {
 
 #define NOT_IMPLEMENTED_EXCEPTION(msg) NotImplementedException(msg, __FILE__, __LINE__)
```

Comment:
We probably want to hold off on porting over the exception system from peloton, since we are still discussing what to do with exceptions in the new system.

Reply:
This is added onto the exceptions stuff that the parser PR had added.